wine-tkg-git/wine-tkg-scripts/build.sh

```diff
@@ -278,10 +278,20 @@ _package_nomakepkg() {
 	cp -v "$_where"/wine-tkg-scripts/wine-tkg-interactive "$_prefix"/bin/wine-tkg-interactive
 
 	# strip
-	if [ "$_pkg_strip" = "true" ] && [ "$_EXTERNAL_INSTALL" != "proton" ]; then
-	  for _f in $( find "$_prefix" -type f '(' -iname '*.dll' -or -iname '*.so' -or -iname '*.sys' -or -iname '*.drv' -or -iname '*.exe' ')' ); do
-	    strip --strip-unneeded "$_f" && msg2 "$_f stripped"
-	  done
+	if [ "$_EXTERNAL_INSTALL" != "proton" ]; then
+	  if [ "$_pkg_strip" = "true" ]; then
+		msg2 "Fixing x86_64 PE files..."
+		find "$_prefix"/"$_lib64name"/ -type f -not '(' -iname '*.pc' -or -iname '*.cmake' -or -iname '*.a' -or -iname '*.la' -or -iname '*.def' -or -iname '*.py' -or -iname '*.pyc' -or -iname '*.pl' ')' -printf '--strip-unneeded\0%p\0%p\0' | xargs -0 -r -P1 -n3 objcopy --file-alignment=4096 --set-section-flags .text=contents,alloc,load,readonly,code
```

Comment:
This "appears" to break strip, wouldn't it be "strip --strip-unneeded" instead of just "--strip-unneeded"?
same for lines 286, 287, 427, 429, 430.

Reply:
It strips the binary via objcopy along with the file alignment and the .text section flags. The -printf part ('--strip-unneeded\0%p\0%p\0') is being added by xargs at the end of the objcopy command.
Proton does the same.

Reply:
Yes, I can see in the [documentation](https://sourceware.org/binutils/docs-2.38/binutils/objcopy.html) that it is possible, interesting that you can use **objcopy** for that, I'm used to **Binutils** for strip